exsim/ltem.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import discretisedfield as df
+import micromagneticmodel as mm
+from scipy import constants
+
+
+# TODO
+# - remove computation of wavelength (not required)
+# - recheck that all units are correct, everything should be in SI units
+# - if possible: remove transposing before/after Fourier transform
+# - Write a proper documentation.
+# - Cs is not used. Can it be removed?
+# - Write tests.
+def ltem_phase(field, /, U, Cs, kx=0.1, ky=0.1):
+    """LTEM phase contrast.
+
+    Parameters
+    ----------
+    field : df.Field
+        Magnetisation field.
+    U : numbers.Real
+        Accelerating voltage of electrons in V.
+    Cs : numbers.Real
+        Spherical aberration coefficient
+    kx : numbers.Real, optional
+        Tikhonov filter radius in x in pixels.
+    ky : numbers.Real, optional
+        Tikhonov filter radius in y in pixels.
+
+    Returns
+    -------
+    df.Field
+        Phase in real space.
+    df.Field
+        Phase in Fourier space.
+
+    Example
+    -------
+    ...
+    """
+    const = 1j * field.mesh.region.edges[2] / (
+        2 * constants.codata.value('mag. flux quantum'))
+    m_sat = np.max(field.norm.array) * mm.consts.mu0
+
+    wavelength = _relativistic_wavelength(U)
+    print(f'The electron beam has a wavelength of {wavelength * 1e9:.2e} nm.')
```

Comment:
Calculating (and printing) the wavelength is not required inside this function and could be removed.